ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ultrassd-disk/provision/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
This file should update?

Reply:
The file is updated , not sure why not showing here - 
```
`[miyadav@centos-strea release]$ git status
On branch azureultrassd
nothing to commit, working tree clean
[miyadav@centos-strea release]$ cat ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/ultrassd-disk/provision/OWNERS
approvers:
- jhou1
- sunzhaohua2
- miyadav
- huali9
reviewers:
- jhou1
- sunzhaohua2
- miyadav
- huali9
`
```